cmake/External/nnpack.cmake

```diff
@@ -35,9 +35,89 @@ endif()
 # (2) Mobile platform - direct build
 ##############################################################################
 
+# We know that NNPACK is only supported with clang due to some builtin
```

Comment:
(this part is for old nnpack clang version. After verifying that gcc works we will remove this.)